src/asound/bluealsa-ctl.c

```diff
@@ -1083,6 +1164,13 @@ SND_CTL_PLUGIN_DEFINE_FUNC(bluealsa) {
 		return -EINVAL;
 	}
 
+	/* single Bluetooth device mode */
+	bool single_device_mode = (bacmp(&ba_addr, BDADDR_ALL) != 0);
+
+	/* non-dynamic operation requires single device mode. */
```

Comment:
Do we really need this constraint? Maybe someone would like to use non-dynamic with default mode. Or maybe non-dynamic with default mode does not make any sense... :D Hmm...

Reply:
I guess it's not really necessary, we should allow each user to make her own choice. However, in that case we probably need separate alsa config default keys for default mode dynamic and single-device mode dynamic, since I think it will be common for users to require dynamic on for default mode and dynamic off for single-device. Or perhaps there should be no config key for the DYN arg and each CTL must explicitly disable it if required?

Reply:
I think that we can change that later, if anyone will complain.